plexpy/webserve.py

```diff
@@ -552,7 +552,7 @@ def library_watch_time_stats(self, section_id=None, **kwargs):
 
         if section_id:
             library_data = libraries.Libraries()
-            result = library_data.get_watch_time_stats(section_id=section_id)
+            result = library_data.get_watch_time_stats(section_id=section_id, time_queries=[1, 7, 30, 0])
```

Comment:
Can leave out the `time_queries` argument since the default is added to `get_watch_time_stats`.